README.md

```diff
@@ -1 +1,32 @@
-# titan
\ No newline at end of file
+# Titan: A RocksDB Plugin to Reduce Write Amplification
+
+Titan is RocksDB Plugin for key-value separation, inspired by 
+[WiscKey](https://www.usenix.org/system/files/conference/fast16/fast16-papers-lu.pdf).
+For introduction and design details, see our
+[blog post](https://pingcap.com/blog/titan-storage-engine-design-and-implementation/).
+
+## Build and Test
+Titan rely on RocksDB source code to build. You need to checkout RocksDB source code locally,
```

Comment:
can we use git submodule instead of local RocksDB?

Reply:
For use in rust-rocksdb repo, i'll make Titan a submodule. Since we already have the rocksdb submodule there, having another nested rocksdb inside titan submodule sounds duplicated. I have been debating with myself with some of the options:
1. require a local RocksDB copy.
2. copy rocksdb internal headers into include/ directory. The downside is we would need to maintain those headers to be consistent with rocksdb.
3. make rocksdb a submodule of Titan, and remove the rocksdb submodule from rust-rocksdb. The downside is whenever we update rocksdb there's one more step to do (i.e. update in pingcap/rocksdb repo->update in titan repo->and update in rust-rocksdb repo).

Which do you think will work better?

Reply:
okay, let's keep this way